openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -220,11 +221,20 @@ def _get_taxa_details(result):
     return taxa_list, title
 
 
+def _get_filesize(img_url):
+    filesize = requests.head(img_url).headers.get("content-length", "")
+    try:
+        return int(filesize)
```

Comment:
I think if this returns a string, you can use [`str::isdigit`](https://docs.python.org/3/library/stdtypes.html#str.isdigit) here rather than a try/except block for doing this conversion!